bc/core/utils/messages.py

```diff
@@ -0,0 +1,3 @@
+NEW_FILING_TEMPLATE = (
+    "New filing in {docket}: {desc} \n\nDoc. on CourtListener: {link}"
+)
```

Comment:
Since mastodon lets us have longer posts, a few things:

1. Let's include a link to the docket.
2. Let's use the link that redirects to the PDF (I think people prefer the PDF).
3. I tweaked the variable name

So:

```suggestion
MASTODON_POST_TEMPLATE = (
    """New filing in {docket}: {desc}

PDF: {pdf_link}
Docket: {docket_link}"""
)
```

Reply:
This is how the new template looks:

![image](https://user-images.githubusercontent.com/55959657/218617177-e3a7f1ad-66ee-46ac-89c4-7743d0c17db0.png)